src/main/java/com/example/learninghub/submit/SubmitController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.learninghub.submit;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/v1/submit")
+public class SubmitController {
+
+    private final SubmitService submitService;
+
+    @Autowired
+    public SubmitController(SubmitService submitService) {
+        this.submitService = submitService;
+    }
+
+    @GetMapping
+    public String get() {
```

Comment:
We should already return correct values from endpoint of this type and also it should take parameter from the path, because right now it wouldn't be useful for testing anything.

Reply:
It's implemented in another branch, and will be added as soon as this is merged